pkgs/applications/video/obs-studio/default.nix

```diff
@@ -50,20 +53,20 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "obs-studio";
-  version = "29.0.2";
+  version = "29.1.3";
 
   src = fetchFromGitHub {
     owner = "obsproject";
     repo = "obs-studio";
     rev = version;
-    sha256 = "sha256-TIUSjyPEsKRNTSLQXuLJGEgD989hJ5GhOsqJ4nkKVsY=";
+    sha256 = "sha256-D0DPueMtopwz5rLgM8QcPT7DgTKcJKQHnst69EY9V6Q=";
     fetchSubmodules = true;
   };
 
   patches = [
     # Lets obs-browser build against CEF 90.1.0+
     ./Enable-file-access-and-universal-access-for-file-URL.patch
-    ./Provide-runtime-plugin-destination-as-relative-path.patch
+    ./fix-nix-plugin-path.patch
```

Comment:
Could you please include a comment here? https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-patch-comment.md

Reply:
I don't agree. The patch is aptly named. No need for further comments.